code/datums/gamemodes/mixed_rp.dm

```diff
@@ -2,8 +2,10 @@
 	name = "mixed (mild)"
 	config_tag = "mixed_rp"
 	latejoin_antag_compatible = 1
-	latejoin_antag_roles = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_WRESTLER, ROLE_ARCFIEND)
-	traitor_types = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_SPY_THIEF, ROLE_ARCFIEND)
+	 //went with a trivial solution of adding more identical items to the list
+	 //Input needed here
+	latejoin_antag_roles = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_WRESTLER, ROLE_ARCFIEND, ROLE_TRAITOR)
+	traitor_types = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_SPY_THIEF, ROLE_ARCFIEND, ROLE_TRAITOR)
```

Comment:
Changing to an associative list and using weighted_pick proc would be a solution